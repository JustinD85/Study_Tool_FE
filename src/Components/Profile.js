import React, { Component } from 'react';

class Profile extends Component{


  render() {
    return <h1>{this.props.person.firstName +`'s Profile`}</h1>
  }
}


export default Profile;