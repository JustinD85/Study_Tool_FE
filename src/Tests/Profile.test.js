import React from 'react';
import Profile from '../Components/Profile';
import { shallow } from 'enzyme';

describe('<Profile/>', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Profile />
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});