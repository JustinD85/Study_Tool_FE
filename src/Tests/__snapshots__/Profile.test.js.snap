// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile/> should match the snapshot 1`] = `
<main
  className="Profile"
>
  <div
    className="profile-user-info animate-info"
  >
    <p>
      Justin
    </p>
    <p>
      Duncan
    </p>
    <p>
      me@me.com
    </p>
  </div>
  <section
    className="profile-javascript animate-info"
  >
    <h1>
      Javascript
    </h1>
    <progress
      max={88}
      value={1}
    />
    <p>
      You have mastered: 1 
                out of  88  words.
    </p>
  </section>
  <section
    className="profile-mastery"
  >
    Vocabulary Mastery!
  </section>
  <section
    className="profile-css animate-info"
  >
    <h1>
      CSS
    </h1>
    <progress
      max={91}
      value={0}
    />
    <p>
      You have mastered: 0
               out of 91 words.
    </p>
  </section>
  <section
    className="profile-html animate-info"
  >
    <h1>
      HTML
    </h1>
    <progress
      max={103}
      value={0}
    />
    <p>
      You have mastered: 0
               out of 103 words.
    </p>
  </section>
  <section
    className="profile-turing animate-info"
  >
    <h1>
      Turing
    </h1>
    <progress
      max={41}
      value={0}
    />
    <p>
      You have mastered: 0
               out of 41 words.
    </p>
  </section>
</main>
`;
